src/tools/projects.md

```diff
@@ -74,7 +74,7 @@ See also [`firefox-data-docs`][docs] for documentation on datasets.
 | Name and repo                     | Description                             |
 |-----------------------------------|-----------------------------------------|
 | [`telemetry.mozilla.org`][tmo_gh] | Main entry point for viewing [aggregate Telemetry data][tmo]
-| [Growth & Usage dashboard][gud]   | Dashboard for questions about product growth and usage
+| [Growth & Usage dashboard][gud]   | Dashboard for questions about product growth and usage ([source][gud_source])
```

Comment:
```suggestion
| [Growth & Usage dashboard][gud_gh]   | Dashboard for questions about [product growth and usage][gud])
```

Switching these around better matches existing style and the headings of this table.